stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/database/internal/ChatDatabase.kt

```diff
@@ -67,7 +67,7 @@ import io.getstream.chat.android.offline.repository.domain.user.internal.UserEnt
         CommandInnerEntity::class,
         SyncStateEntity::class,
     ],
-    version = 69,
+    version = 70,
```

Comment:
If I got it right we've increased the version here due `Date` nullability changes inside `ChannelUserReadEntity`, is that right? 🙂

Reply:
Yep, It is needed because DB Structure is different